OpenRA.Mods.Common/Widgets/Logic/AssetBrowserLogic.cs

```diff
@@ -129,7 +129,7 @@ public AssetBrowserLogic(Widget widget, Action onExit, ModData modData, WorldRen
 			var spriteWidget = panel.GetOrNull<SpriteWidget>("SPRITE");
 			if (spriteWidget != null)
 			{
-				spriteWidget.GetSprite = () => currentSprites?[currentFrame];
+				spriteWidget.GetSprite = () => currentSprites?.Length > 0 ? currentSprites[currentFrame] : null;
```

Comment:
```suggestion
				spriteWidget.GetSprite = () => currentSprites != null && currentSprites.Length > 0 ? currentSprites[currentFrame] : null;
```

This feels like better formatting

Reply:
I feel #20680 will not agree with that though.

Reply:
It does, when I change the style it doesn't give a suggestion